docker/appserver/WebSphere/src/scripts/start-with-mssql.bat

```diff
@@ -1,4 +1,4 @@
 cd %~dp0/../..
 docker container stop iaf-test-as-websphere-with-mssql
 docker container rm iaf-test-as-websphere-with-mssql
-docker run --publish 9443:9443 --publish 9043:9043 --mount type=bind,source="%~dp0/../../../../../../test/src/test/testtool",target=/opt/frank/testtool-ext -e jdbc.dbms.default=mssql --name iaf-test-as-websphere-with-mssql iaf-test-as-websphere
\ No newline at end of file
+docker run --publish 80:9080 --publish 9443:9443 --publish 9043:9043 --mount type=bind,source="%~dp0/../../../../../../test/src/test/testtool",target=/opt/frank/testtool-ext -e jdbc.dbms.default=mssql --name iaf-test-as-websphere-with-mssql iaf-test-as-websphere
```

Comment:
Waarom moet de http poort beschikbaar komen? De tests werken ook zonder.
Waarom wordt hij gemapt op poort 80? (de https poort wordt ook niet gemapt op poort 443)

Reply:
Bij Tomcat en Jboss zetten we ook poort 80 open in de start*.bat-scripts.